tfhe/Cargo.toml

```diff
@@ -324,3 +330,10 @@ required-features = ["integer", "internal-keycache"]
 
 [lib]
 crate-type = ["lib", "staticlib", "cdylib"]
+
+[lints.rust]
+unexpected_cfgs = { level = "warn", check-cfg = [
+    'cfg(__profiling)',
+    'cfg(bench)',
```

Comment:
bench is not whitelisted by default ? 🤔

Reply:
can you see if there has been an update to cfgs for bench ? otherwise I would report this one as a false positive to the rust/clippy projects

Maybe some people already reported this

Reply:
What do you mean by an update?

Reply:
i.e. that bench is/was a legit cfg but it's now missing, has it been complitely removed or is it some sort of mistake on the clippy side of things ?